std/range/package.d

```diff
@@ -7419,11 +7419,11 @@ For `windowSize = 1` it splits the range into single element groups (aka `unflat
 For `windowSize = 2` it is similar to `zip(source, source.save.dropOne)`.
 
 Params:
-    f = If `Yes.slidesWithLessElements` slides with fewer
+    f = If `Yes.slideWithLessElements` slide with fewer
```

Comment:
Looks like we also should do s/slideWithLessElements/slideWithFewerElements/g or better yet (because it's obvious from context) s/slideWithLessElements/withFewerElements/g